src/tree/LocalRootTreeItemBase.ts

```diff
@@ -89,8 +95,12 @@ export abstract class LocalRootTreeItemBase<TItem extends ILocalItem, TProperty
                 const refreshInterval: number = this.getRefreshInterval();
                 intervalId = setInterval(
                     async () => {
-                        if (window.state.focused && await this.hasChanged()) {
-                            await this.refresh();
+                        if (this.autoRefreshEnabled && await this.hasChanged()) {
```

Comment:
Not really important, and it admittedly looks a little odd, but I believe you can do this to avoid another `if` inside:
```suggestion
                        if (this.autoRefreshEnabled && await this.hasChanged() && this.autoRefreshEnabled) {
```
I'd definitely keep the comment explaining why if we do it that way

Reply:
Indeed that looks odd. I think another if is more readable.

Reply:
I think what Ravi has here is good enough, no need to be too clever